docker/ONNXruntime/Dockerfile

```
@@ -0,0 +1,124 @@
+        -O /tmp/openvino.tgz --no-check-certificate --quiet && \
+    tar -zxvf /tmp/openvino.tgz -C /opt/intel/ && \
+    sudo ln -s /opt/intel/l_openvino_toolkit_ubuntu20_${OV_LONG_VERSION} $INSTALL_DIR && \
+    rm -rf /tmp/*
+
+# Install OpenVINO dependencies
+WORKDIR $INSTALL_DIR/install_dependencies
+RUN ls -la $INSTALL_DIR/install_dependencies && echo y | ./install_openvino_dependencies.sh
+RUN cat /root/.bashrc > tmp && echo 'source $INSTALL_DIR/setupvars.sh' > /root/.bashrc
+
+# Install openvino-dev
+WORKDIR /tmp/
+RUN pip3 install --upgrade pip
+RUN pip3 install openvino-dev==${OV_SHORT_VERSION}
+
+# Install OpenCV
+ARG OCV_VERSION=4.7.0
+RUN pip uninstall opencv-python -y
+RUN git clone --recurse-submodules https://github.com/opencv/opencv.git  --depth 1 --branch ${OCV_VERSION} --single-branch
+ENV OpenCV_BUILD_DIR=/root/build-opencv
+RUN mkdir $OpenCV_BUILD_DIR
+WORKDIR $OpenCV_BUILD_DIR
+RUN /bin/bash -c 'source $INSTALL_DIR/setupvars.sh && \
+    cmake -G Ninja \
+        -D CMAKE_INSTALL_PREFIX=install \
+        -D CMAKE_BUILD_TYPE=Release \
+        -D BUILD_EXAMPLES=OFF \
+        -D BUILD_TESTS=OFF \
+        -D BUILD_DOCS=OFF \
+        -D WITH_OPENVINO=ON \
+        -D OPENCV_LIB_INSTALL_PATH=lib \
+        -D OPENCV_CONFIG_INSTALL_PATH=cmake \
+        -D PYTHON3_PACKAGES_PATH=install/python/python3 \
+        /tmp/opencv/ && ninja && cmake --install .' && \
+    rm -rf /tmp/*
+ENV OpenCV_INSTALL_DIR="$OpenCV_BUILD_DIR/install"
+ENV OpenCV_DIR="$OpenCV_INSTALL_DIR/cmake"
+ENV LD_LIBRARY_PATH="$OpenCV_INSTALL_DIR/lib${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}"
+ENV PYTHONPATH="$OpenCV_INSTALL_DIR/python/python3/cv2/python-3.8${PYTHONPATH:+:$PYTHONPATH}"
+
+#install onnxruntime
+WORKDIR /tmp/
+ARG ONNX_VERSION=1.14.0
+RUN git clone https://github.com/microsoft/onnxruntime.git && \ 
+    cd onnxruntime && \ 
+    git checkout v${ONNX_VERSION} && \ 
+    git submodule update --init --recursive && mkdir build && cd build && \
+    cmake -DCMAKE_INSTALL_PREFIX=install \
+        -DCMAKE_BUILD_TYPE=Release \
+        -Donnxruntime_BUILD_FOR_NATIVE_MACHINE=ON \
+        -Donnxruntime_BUILD_UNIT_TESTS=OFF \
+        -Donnxruntime_BUILD_SHARED_LIB=ON \
+        -Donnxruntime_USE_FULL_PROTOBUF=ON ../cmake && \ 
+    make install -j$(nproc --all)
+RUN pip3 install onnxruntime==${ONNX_VERSION}
+
+# accuracy-check
+ARG OMZ_VERSION="${OV_SHORT_VERSION}.0"
+WORKDIR /tmp/
+RUN git clone --recursive https://github.com/openvinotoolkit/open_model_zoo.git --branch ${OMZ_VERSION} --single-branch --depth 1
+WORKDIR /tmp/open_model_zoo/tools/accuracy_checker
+RUN wget https://www.cs.toronto.edu/~kriz/cifar-10-python.tar.gz
+RUN tar xvf cifar-10-python.tar.gz -C sample
+RUN /bin/bash -c 'source $INSTALL_DIR/setupvars.sh && \
+    accuracy_check -c sample/sample_config.yml -m data/test_models -s sample'
```

`openvino-dev` включает в себя инструменты для DL, в том числе `AccuracyChecker`. Это как отдельный инструмент, которому `setupvars.sh` не нужен. Он должен автоматически запускаться "из коробки" (из pip пакета). Я это к тому, что можно написать следующим образом:
```suggestion
RUN  accuracy_check -c sample/sample_config.yml -m data/test_models -s sample
```

Кроме того, вы на 63 строке добавляется запуск скрипта `setupvars.sh` непосредственно в `/root/.bashrs`, что позволяет не писать каждый раз `/bin/bash -c 'source $INSTALL_DIR/setupvars.sh ...`, поскольку команда `/bin/bash` самостоятельно будет запускать этот скрипт. Соответственно можно просто писать так (но вообще, как я написала выше, вам нужно просто сразу из питона запустить):
```
RUN /bin/bash -c "accuracy_check -c sample/sample_config.yml -m data/test_models -s sample"
```